doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1482,16 +1512,8 @@ Chapter :ref:`Tactics`. The basic assertion command is:
       This feature, called nested proofs, is disabled by default.
       To activate it, turn the :flag:`Nested Proofs Allowed` flag on.
 
-   .. cmdv:: Lemma @ident {? @binders } : @type
-             Remark @ident {? @binders } : @type
-             Fact @ident {? @binders } : @type
-             Corollary @ident {? @binders } : @type
-             Proposition @ident {? @binders } : @type
-      :name: Lemma; Remark; Fact; Corollary; Proposition
-
-      These commands are all synonyms of :n:`Theorem @ident {? @binders } : type`.
-
-.. cmdv:: Theorem @ident {? @binders } : @type {* with @ident {? @binders } : @type}
+.. cmd:: thm_token @ident {? @binders } : @type {* with @ident {? @binders } : @type}
```

Comment:
This used to be a `cmdv`. Now that it is a `cmd`, it should be moved below the `Definition` and `Let` variants, just above the `Fixpoint` and `CoFixpoint` variants.

Reply:
Mixed feelings about this.  You wanted to split this in 2, only showing the `with` clause in the second one.  But it's really good to keep all the syntax together.  And I'd like to get rid of all the cmdvs, if possible.  Take a look at my latest change--do you think that's workable